.nycrc

```diff
@@ -1,6 +1,6 @@
 {
-    "temp-dir" : "build/.coverage",
-    "report-dir" : "build/.coverage-report",
+    "temp-dir" : "coverage-temp",

```

Comment:
What are the pros/cons of putting the coverage files in '.' vs 'build'?

Reply:
This may have been done without the context I needed. I saw `build` and assumed it was intended to be the output of `tsc` and the entry point for the tool. Looking at `package.json` now, I can see you specify `build/src` for `main`, `types` and `files`. I'm fine with either. In most projects I've done in the past, `tsc` outputs to something like `dist` or `lib` and then these temporary build folders are kept at root and ignored. We can chat about this offline if you'd like.